package.json

```diff
@@ -180,5 +180,8 @@
     "webpack": "^4.30.0",
     "webpack-cli": "^3.3.2",
     "webpack-dev-server": "^3.3.1"
+  },
+  "dependencies": {
+    "@skyscanner/backpack-web": "^34.10.1"
```

Comment:
This isn't to have been installed can you remove?

Reply:
Oh, I missed this. Done now thanks!